backend/server.py

```
@@ -181,18 +266,25 @@ def get_boolean_answer():
+        document_url = data.get("document_url")
         if not document_url:
-            return jsonify({'error': 'Document URL is required'}), 400
+            return jsonify({"error": "Document URL is required"}), 400
 
         text = docs_service.get_document_content(document_url)
         return jsonify(text)
+
     except ValueError as e:
-        return jsonify({'error': str(e)}), 400
+        return jsonify({"error": str(e)}), 400
     except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)